test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -1,5 +1,7 @@
 using namespace System.Diagnostics
 
+Import-Module $PSScriptRoot\..\Common\Test.Helpers.psm1
```

Comment:
in what way is our structure not intuitive? In what way is our structure too deep? I certainly agree that our tools should not be hidden